packages/gatsby-source-filesystem/README.md

```diff
@@ -130,7 +130,7 @@ createFilePath({
 
 #### Example usage
 
-The following is taken from [Gatsby Tutorial, Part Seven](https://www.gatsbyjs.org/tutorial/part-7/) and is used to create URL slugs for markdown pages.
+The following is taken from [Gatsby Tutorial, Part Seven](https://www.gatsbyjs.com/tutorial/part-7/) and is used to create URL slugs for markdown pages.
```

Comment:
This is a dead link and it appears the "copied" content is from the old tutorial

Reply:
Not sure if the Tutorial part 7 content is still the same/fits the context,but it's not at `/docs/tutorial…`: https://www.gatsbyjs.com/docs/tutorial/part-7/

Reply:
It's not the same context, I'm just going to opt to remove this reference in the README.